pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -47,7 +47,7 @@ func validateBrokerSpec(spec *sc.BrokerSpec, fldPath *field.Path) field.ErrorLis
 
 	if "" == spec.URL {
 		allErrs = append(allErrs,
-			field.Required(fldPath.Child("URL"),
+			field.Required(fldPath.Child("url"),
```

Comment:
Normally I would be totes onboard with that, but i'm not sure that makes things more readable in this case.